src/bare-metal/no_std.md

```diff
@@ -0,0 +1,65 @@
+# `no_std`
+
+<table>
```

Comment:
Have you considered ditching the table and instead use three slides, one for each crate?

I used to have a table myself for some other part of the course, but a linear list with headings proved to be much simpler to read at a glance. The advantage of such a list is that you can slowly scroll up and down:

![image](https://user-images.githubusercontent.com/89623/225897228-938f24c2-03c8-41ef-a523-6311b72a2f8c.png)

Reply:
I had a list before, but I decided to go with a table so that I can have everything on one screen and discuss the distinctions without having to scroll or switch pages back and forth.

Reply:
This is my attempt at a top-level summary. The module names aren't necessarily entirely obvious, and often there are modules with the same name split across `core`/`alloc`/`std` with different contents, so I think it's more useful to give examples of commonly used items from the modules than the modules themselves. My plan is to spend a while on this page talking through the distinctions and the reasons for them, and having to keep switching back and forth would be distracting. I don't expect students to read everything, but being able to spot a few things they have used before and see where they fall is useful.

Reply:
To give some examples of why module names aren't enough: `collections::BinaryHeap` is in `alloc`, but `collections::HashMap` is only in `std`. `fmt::write!` is in `core`, but `fmt::format!` is in `alloc`.

Reply:
> My plan is to spend a while on this page talking through the distinctions and the reasons for them

Okay, I would love some details about this in the speaker notes so that someone (like me!) can say something smart about the page. Otherwise I would probably go over it in much too little time. I'll of course also take notes when we do a trial run of this new material.